pkg/util/options/parser.go

```diff
@@ -0,0 +1,71 @@
+package options
```

Comment:
Do you mind if we put this as a package in `pkg/provider/terraform`?    The options parsing here is pretty specific to terraform at the moment so it may make sense to have this code as a package within terraform.

Reply:
Sine we are using the native templating from the plugin config file this entire package is unnecessary; I will remove.